.github/workflows/rust.yml

```diff
@@ -79,12 +79,47 @@ jobs:
           ips=$(kubectl get nodes -lkubernetes.io/hostname!=kind-control-plane -ojsonpath='{.items[*].status.addresses[?(@.type=="InternalIP")].address}')
           port=$(kubectl -n default get service kuard -ojsonpath='{.spec.ports[?(@.name=="envoy-http")].nodePort}')
           curl "http://${ips[0]}:${port}"
+
+  image:
+    name: Build docker image
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - dockerfile: Dockerfile
+            platform: linux/amd64
+            scope: build-amd
+            artifactsuffix: amd
+          - dockerfile: Dockerfile.aarch64
+            platform: linux/arm64
+            scope: build-arm
+            artifactsuffix: arm
```

Comment:
nit: these are not needed here since this test just builds the image and doesn't output to manifests or github cache

Reply:
Fixed. 

Ready for review  again.